src/App.js

```diff
@@ -1,13 +1,25 @@
 import { data } from "./db.js";
 import "./App.css";
 import AlbumList from "./components/AlbumList/Index.js";
+import { useState, useEffect } from "react";
 
 export default function App() {
+  const [data, setData] = useState([]);
+
+  useEffect(() =>{
+    async function fetchAlbumData() {
+      const response = await fetch("https://neuefische-spotify-proxy.vercel.app/api/featured");
+      const albumData = await response.json();
+      setData(albumData);
```

Comment:
You might want to consider adding a try...catch here incase something goes wrong with the request.